lib/shopify_app/sessions_controller.rb

```diff
@@ -16,6 +16,7 @@ def callback
         sess = ShopifyAPI::Session.new(shop_name, response['credentials']['token'])
         session[:shopify] = ShopifyApp::SessionRepository.store(sess)
         session[:shopify_domain] = shop_name
+        session[:return_to] = nil
```

Comment:
yeah but if return_to was actually set now we have unset and not used it.

I think the solution here is just to clean this up after we've used it is all.